scripts/install-mender.sh

```diff
@@ -202,19 +202,29 @@ get_deps() {
 add_repo() {
```

Comment:
This function is what I have modified in the last force-push. @merlin-northern @tranchitella could you please review it?